components/ButtonAvatar.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { TouchableOpacity, Image } from 'react-native'
+import { styles } from './ButtonAvatar.styles'
+
+export const AvatarCircle = (props) => {
```

Comment:
do we use that somewhere else than in this file ? If not we don't need to export it, right?